api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -369,11 +369,13 @@ const (
 	PruneRetentionPolicy RetentionPolicy = "Prune"
 )
 
-// +kubebuilder:validation:Enum=ImageRegistry
+// +kubebuilder:validation:Enum=ImageRegistry;openshift-samples
```

Comment:
We should be consistent in the naming convention. If we already used camelcase for ImageRegistry, this should be OpenShiftSamples